ft_prepare_sourcemodel.m

```diff
@@ -725,6 +725,44 @@
   end
 end
 
+if strcmp(cfg.movetocentroid, 'yes')
```

Comment:
right now the cfg` does not have the `movetocentroids` field. You should add that somewhere around line 150